backend/song/api/urls.py

```diff
@@ -0,0 +1,45 @@
+from django.urls import path
+
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from backend.song.api.views import (
+    CategoriesList,
+    CategoriesDetail,
+    ArtistsList,
+    ArtistsDetail,
+    AlbumsList,
+    AlbumsDetail,
+    SongsList,
+    SongsDetail,
+    LanguagesList,
+    LanguagesDetail,
+    LyricsList,
+    LyricsDetail,
+    LyricRequestsList,
+    LyricRequestsDetail,
+    TranslationsList,
+    TranslationsDetail
+)
+
+app_name = 'song'
+
+urlpatterns = [
+    path('categories/', CategoriesList.as_view()),
+    path('categories/<int:pk>/', CategoriesDetail.as_view()),
+    path('artists/', ArtistsList.as_view()),
```

Comment:
You can add the name of the URLs here so that you will not need to hardcode the URLs in the test or later in the HTML templates when calling them...